src/Features/Core/Portable/ExternalAccess/UnitTesting/API/UnitTestingIncrementalAnalyzerProvider.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.SolutionCrawler;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
```

Comment:
Should this be outside the Api namespace?
```suggestion
namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting
```

Reply:
No, this is intentionally an API that UnitTesting will consume.

Reply:
I thought we only intended to expose interfaces to partners, not concrete types. Did that design decision change?

Reply:
I am not aware of any such rule.

Reply:
Not sure why would that be needed.